code/modules/reagents/chemistry/recipes/toxins.dm

```diff
@@ -92,8 +86,8 @@
 	name = "Rotatium"
 	id = "Rotatium"
 	results = list("rotatium" = 3)
-	required_reagents = list("mindbreaker" = 1, "teslium" = 1, "neurotoxin2" = 1)
-	mix_message = "<span class='danger'>After sparks, fire, and the smell of mindbreaker, the mix is constantly spinning with no stop in sight.</span>"
+	required_reagents = list("lsd" = 1, "teslium" = 1, "neurotoxin2" = 1)
+	mix_message = "<span class='danger'>After sparks, fire, and the smell of LSD, the mix is constantly spinning with no stop in sight.</span>"
```

Comment:
It has a smell, but no taste?

Reply:
LSD tastes of "sourness" so I suppose this can too.

Reply:
LSD tastes of `"sourness"`. Rotatium tastes of `"spinning"`.

Reply:
Huh I could have sworn there wasn't a taste when I looked.